### ISSUE TITLE
<h1>Still not working</h1>

### ISSUE THREAD

**Issue by tushar260** [wasdennnoch/AndroidN-ify#982]
![screenshot_20160927-102441](https://cloud.githubusercontent.com/assets/21682756/18860639/ed004122-849c-11e6-84a5-92f2eea0b5d1.png)
![screenshot_20160927-102451](https://cloud.githubusercontent.com/assets/21682756/18860645/f8b335f6-849c-11e6-8c61-2e3dce1fa4bd.png)

**Comment by paphonbth** [comment]
Some apps post notifications with public variant. It won't be considered sensitive in that case.

**Comment by tushar260** [comment]
Public variant as in?

**Comment by tushar260** [comment]
The lock/unlock button has no effect on their sensitivity.

**Comment by paphonbth** [comment]
Did you try other app? That button has the same effect as the one in lock screen settings. Just app-specific.

**Comment by tushar260** [comment]
That option in lock screen settings is no more there. (Show all notifications, hide sensitive notification..)

**Comment by tushar260** [comment]
Heyyy. I got that working 
I just remove security and then reapplied PIN and then the option appeared (show all, hide sensitive) and in that I selected show all notifications. That's the correct way as when we select show sensitive only, we're not allowed to swipe down without entering pin/password. 

Great great great work bro! Thank you so very much :D 
I wanted this since the inception of this option! Thanks again :))
